content/en/docs/collector/custom-collector.md

```diff
@@ -20,13 +20,12 @@ builder][ocb] (or `ocb` for short) to assist people in assembling their own
 distribution, making it easy to build a distribution that includes their custom
 components along with components that are publicly available.
 
-As part of the process the `ocb` will generate the Collector's source code,
-which you can use to help build and debug your own custom components, so let's
-get started.
+As part of the process, the `ocb` generates the Collector's source code,
+which you can use to build and debug your own custom components.
 
 ## Step 1 - Install the builder
 
-{{% alert color="primary" title="Note" %}}
+{{% alert title="Note" %}}
```

Comment:
I think we should pull this content out of an admonition and make a new section called `## Prerequisites` that comes before the steps.